sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,59 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value",
+                         [(0, 100), (None, None), (-np.inf, np.inf),
+                          ((-5, 5, 10), (100, 200, 300)),
+                          ((-5, None, 10), (100, 200, None))])
+def test_iterative_imputer_min_max_completion(min_value, max_value):
+    # Test for issue #16349 which adds the capability
```

Comment:
Since this is not a regression, I think this is better to have 

```
# check that passing scalar or array-like works
```

Reply:
Done!